deployments/run.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Script used for running and updating bridge deployments.
+#
+# To deploy a network you can run this script with the name of the network you want to run.
+#
+# `./run.sh eth-poa-sub`
+#
+# To update a deployment to use the latest images available from the Docker Hub add the `update`
+# argument after the bridge name.
+#
+# `./run.sh rialto-millau update`
+#
+# Once you've stopped having fun with your deployment you can take it down with:
+#
+# `./run.sh rialto-millau stop`
+
+set -xeu
+
+RIALTO=' -f ./networks/docker-compose.rialto.yml'
+MILLAU=' -f ./networks/docker-compose.millau.yml'
+ETH_POA=' -f ./networks/docker-compose.eth-poa.yml'
+MONITORING=' -f ./monitoring/docker-compose.yml'
+
+BRIDGE=''
+NETWORKS=''
+case "$1" in
+	eth-poa-sub)
```

Comment:
As in some other comment, I think this might rather be two separate bash scripts with a shared part, but not super strong on this, since we only have two bridges now.

Reply:
I don't really see the value in different scripts at this point in time. Especially since the only split in logic is choosing two strings